modules/apps/foundation/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -38,7 +38,6 @@ String onChangeMethod = (String)request.getAttribute("liferay-ui:input-editor:on
 String onFocusMethod = (String)request.getAttribute("liferay-ui:input-editor:onFocusMethod");
 String onInitMethod = (String)request.getAttribute("liferay-ui:input-editor:onInitMethod");
 String placeholder = GetterUtil.getString((String)request.getAttribute("liferay-ui:input-editor:placeholder"));
-String required = (String)request.getAttribute("liferay-ui:input-editor:required");
```

Comment:
Hey @antoniopol06 

Is this a part of the tooltip fix? It seems that might be unrelated